### ISSUE TITLE
Blank input on Atom 0.184.0

### ISSUE THREAD

**Issue by MatthewMueller** [rev087/fancy-new-file#27]
When I use fancy-new-file on the latest atom, I'm not seeing the input.

![](https://cldup.com/zgYLOTIDvR.png)

Doesn't seem to be theme related because it doesn't work on Atom Dark or Atom Light.

**Comment by mhutter** [comment]
The case is probably this deprecation Warning:

> fancy-new-file
> keymaps/fancy-new-file.cson
> /!\ Use the `atom-workspace` tag instead of the `workspace` class.

**Comment by jeffschwartz** [comment]
+1

**Comment by JFGagnon** [comment]
+1

**Comment by sgunturi** [comment]
+1

**Comment by nikosolihin** [comment]
+1

It's actually working but just no text

**Comment by Trudko** [comment]
Hi guys, you might switch to my fork of this package [Advanced New File](https://atom.io/packages/advanced-new-file).

**Comment by MatthewMueller** [comment]
@Trudko works great. thanks man

**Comment by jeffschwartz** [comment]
Thank you for picking up on this. If you get the time please see my suggestion I gave for fancy new file to bring it more in line with the way advanced new file works in sublime text. In sublime text the new file is always created even if it doesn’t contain content - it shows up immediately in the project tree. With your fork and the original the new file is created immediately  and if the new file doesn’t contain content and the focus changes to another file the new file isn’t saved.
-
Jeff

**Comment by Trudko** [comment]
@jeffschwartz yeah I didn't forget about you! Just too busy spending time on blog about Atom. I want to add test to advanced new file before adding anything new, but this might take a while.

**Comment by maksimf** [comment]
Same issue

**Comment by Trudko** [comment]
@maksimf please use advanced new file instead

**Comment by maksimf** [comment]
@Trudko ok, thanks!

**Comment by rhoen** [comment]
Thanks for fixing this bug in advanced new file - this feature is very handy!

**Comment by Trudko** [comment]
@rhoen my pleasure.
